composables/use-error-handler.ts

```diff
@@ -34,12 +34,13 @@ export default function () {
     // Generic error handling
     if (!handler) {
       switch (rawErrorMessage) {
+        case 'INSUFFICIENT_PERMISSION':
         case 'TOKEN_EXPIRED':
           handler = {
             description: $t('error_token_expired'),
-            onClose: () => {
-              accountStore.logout()
-              navigateTo(localeRoute({ name: 'account' }))
+            onClose: async () => {
+              await accountStore.logout()
```

Comment:
Do we need to catch here? Like if `logout()` throws error because we aren't actually logged in or sth